### ISSUE TITLE
RCS1210 is not reported when return type is non-generic task

### ISSUE THREAD

**Issue by cezarypiatek** [JosefPihrt/Roslynator#730]
**Product and Version Used**:  3.0.0
**Affected rule**: https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1210.md

```csharp
Task DoAsync() 
{
    return null; //Expecting RCS1210
}
```

For generic `Task<>`  RCS1210  is working correctly

```cs
Task<object> GetSomethingAsync() 
{
    return null;  // RCS1210 reported correctly
}
```
